src/platform/user/profile/vet360/actions/transactions.js

```diff
@@ -162,6 +162,12 @@ export function createTransaction(
         ? await apiRequest(route, options)
         : await localVet360.createTransaction();
 
+      if (transaction?.errors) {
```

Comment:
pretty sure we don't need the `?` here. if `transaction` doesn't exist at this point we've got problems since we just set it in the previous statement.